docs/site/Repositories.md

```diff
@@ -133,199 +146,195 @@ export class AccountRepository extends DefaultCrudRepository {
+      content: {
+        'application/json': {
+          schema: {
+            type: 'object',
+          },
+        },
       },
-    ],
+    },
     responses: {
       200: {
-        schema: {
-          accountInstance: "#/definitions/Account"
+        description: 'Account instance created',
+        content: {
+          'application/json': {
+            schema: {
+              $ref: '#/components/schemas/Account',
+            },
+          },
         },
       },
     },
   },
 }
 ```
+
 Create Controller method:
-```javascript
-async createAccount(accountInstance) {
+
+```ts
+async createAccount(accountInstance: Account) {
   return await this.repository.create(accountInstance);
 }
 ```
-2. Find API Definition:
-```javascript
+
+2.  Find API Definition:
+
+```ts
 '/accounts': {
   get: {
     'x-operation-name': 'getAccount',
-    parameters: [
-      {
-        name: 'filter',
-        in: 'query',
-        description: 'The criteria used to narrow down the number of accounts returned.',
-        required: false,
-        type: 'object'
-      }
-    ],
     responses: {
       200: {
-        schema: {
-          type: 'array',
-          items: '#/definitions/Account'
+        description: 'List of accounts',
+        content: {
+          'application/json': {
+            schema: {
+              type: 'array',
+              items: {
+                $ref: '#/components/schemas/Account',
+              },
+            },
+          },
         },
       },
     },
   },
 }
 ```
+
 Find Controller method:
-```javascript
-async getAccount(filter) {
-  return await this.repository.find(JSON.parse(filter));
-}
-```
-3. Update API Definition:
-```javascript
-'/accounts/update': {
-  post: {
-    'x-operation-name': 'updateAccount',
-    parameters: [
-      {
-        name: 'where',
-        in: 'query',
-        description: 'The criteria used to narrow down the number of accounts returned.',
-        required: true,
-        type: 'object'
-      },
-      {
-        name: 'data',
-        in: 'body',
-        description: 'An object of model property name/value pairs',
-        required: true,
-        type: 'object'
-      }
-    ],
-    responses: {
-      200: {
-        schema: {
-          type: 'object',
-          description: 'update information',
-          properties: {
-            count: {
-              type: 'number',
-              description: 'number of records updated'
-            }
-          }
-        },
-      },
-    },
-  },
-}
-```
-Update Controller method:
-```javascript
-async updateAccount(where, data) {
-  return await this.repository.update(JSON.parse(where), data);
+
+```ts
+async getAccount() {
+  return await this.repository.find();
 }
 ```
 
-Please See [Testing Your Application](Testing-Your-Application.md) section in order to set up and write unit, acceptance, and integration tests for your application.
+Don't forget to register the complete version of you OpenAPI spec through
```

Comment:
`complete version of you OpenAPI spec` -> `complete version of your OpenAPI spec`?